test/test_float.erl

```diff
@@ -0,0 +1,6 @@
+-module(test_float).
+
+-export([f/0]).
+
+%% test case for float literal
+f() -> 0.1.
```

Comment:
Could you add test cases for exponential notation?

e.g.,

```
7> 2.3e3.
2.3e3
8> 2.3e-3.
0.0023
```

in http://erlang.org/doc/reference_manual/data_types.html